docs/resources/new_in_doc.md

```diff
@@ -6,6 +6,79 @@ description: Overview of major recent additions to Ibexa DXP documentation.
+- Reorganized the [Updating Ibexa DXP](https://doc.ibexa.co/en/latest/update_and_migration/update_ibexa_dxp/) section to put information in logical order and remove duplicates
+- [Added v4.6.11 to v4.6.12 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_4.6/update_from_4.6/#v4612)
+- [Added v4.6.12 to v4.6.13 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_4.6/update_from_4.6/#v4613) mentioning a command to clean up duplicated entries in the `ezcontentobject_attribute` table
+- Changed the [Update to v3.3](https://doc.ibexa.co/en/latest/update_and_migration/from_2.5/to_3.3/#b-update-the-app) instructions to help avoid an error at `composer update` stage
+- Changed the instructions by adding a requirement to handle the [[= product_name_cloud =]] configuration:
+    - [Update to v4.0](https://doc.ibexa.co/en/master/update_and_migration/from_3.3/to_4.0/#ibexa-cloud)
+    - [Update to v3.3.latest](https://doc.ibexa.co/en/master/update_and_migration/from_3.3/update_from_3.3/#v3313)
+- Added a suggestion to [remove obsolete database tables](https://doc.ibexa.co/en/master/update_and_migration/from_4.3/update_from_4.3_new_commerce/#update-the-database) that were used by a legacy Commerce package
+
+## Personalization
+
+- Added dynamic attribute submodels information by:
+    - mentioning them in [customizing the recommendation request](https://doc.ibexa.co/en/master/personalization/api_reference/recommendation_api/#customizing-the-recommendation-request) instructions
+    - describing them in [user documentation](https://doc.ibexa.co/projects/userguide/en/latest/personalization/recommendation_models/#dynamic-attributes)
+- Added time-slot based models information by:
+    - changing the list of parameters available when [customizing the recommendation request](https://doc.ibexa.co/en/master/personalization/api_reference/recommendation_api/#customizing-the-recommendation-request)
+    - describing them in [user documentation](https://doc.ibexa.co/projects/userguide/en/master/personalization/recommendation_models/#time-slot-based-models)
+
+- Updated configuration details (including endpoint addresses nad code examples) in multiple how-to articles:
+    - [Enable Personalization ](https://doc.ibexa.co/en/master/personalization/enable_personalization)
+    - [Integrate recommendation service](https://doc.ibexa.co/en/master/personalization/integrate_recommendation_service/)
+    - [Tracking integration](https://doc.ibexa.co/en/master/personalization/tracking_integration/)
+    - [Track events with ibexa-tracker.js](https://doc.ibexa.co/en/master/personalization/tracking_with_ibexa-tracker/)
+
+### PIM
+
+- Updated the [Product API](https://doc.ibexa.co/en/master/pim/product_api/) article by fixing method signatures and adding links to the PHP API reference
+
+### PHP API
+
+- Added missing classes in the following namespaces:
```

Comment:
```suggestion
- The following new classes are now part of the public PHP API:
```
The classes were introduced with a bugfix release of Ibexa DXP (didn't exist before, exist now), so I'm not sure about calling them "missing"